packages/core/src/odata-v4/request-builder/create-request-builder.spec.ts

```diff
@@ -47,6 +47,8 @@ describe('CreateRequestBuilderV4', () => {
     testPostRequestOutcome(actual, entity.setOrInitializeRemoteState());
   });
 
+  // TODO: This test is quite useless and wrong. It does not fail even if we remove the multilink.
```

Comment:
This is not a nice comment, we should remove it ;)

Reply:
@jjtang1985 I am not sure what to do with this file. To be honest it seems to me that all the tests are quite useless. We are basically checking that the deserializer works. Which is already checked in the deserialize

Reply:
Would you fix this test or make a follow-up? 😄

Reply:
The question is - what should be tested here? The v4 create request builder doesn't do anything, therefore I would say it is too trivial to be tested.

Reply:
I now removed the test file, but also created a follow up ticket to ensure our odata tests are complete and correct ;)